pkg/meta/base.go

```diff
@@ -705,6 +723,9 @@ func (m *baseMeta) checkDirQuota(ctx Context, inode Ino, space, inodes int64) bo
 }
 
 func (m *baseMeta) updateDirQuota(ctx Context, inode Ino, space, inodes int64) {
+	if !m.GetFormat().DirStats {
```

Comment:
We should enable it when first quota was set.

Reply:
done